test/parallel/test-whatwg-webstreams-compression.js

```diff
@@ -51,17 +51,21 @@ Promise.all(['gzip', 'deflate'].map((i) => test(i))).then(common.mustCall());
 
 assert.throws(
   () => Reflect.get(CompressionStream.prototype, 'readable', {}), {
-    code: 'ERR_INVALID_THIS',
+    name: 'TypeError',
+    message: /Cannot read private member/,
```

Comment:
I suggest not to validate the error message as this is in control of V8, and make upgrading V8 more work than necessary.

Reply:
I agree, but this was a recommendation from @jasnell at https://github.com/nodejs/node/pull/46904#discussion_r1122478152.